src/style.css

```
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
+}
+
+.stars:nth-child(n + 2) {
+  margin-top: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(n + 2) {
+  margin-left: 4px;
+}
+
+.stars--1 :nth-child(-n + 1) {
```

```suggestion
.stars--1 :nth-child(-n + 1),
... ,
.stars--5 :nth-child(-n + 5) {
```
Consider using commas to group selectors